artman/tasks/protoc_tasks.py

```diff
@@ -519,3 +519,24 @@ def execute(self, gapic_api_yaml):
                 continue
             if 'package_name' in go_settings:
                 return go_settings.get('package_name')
+
+
+class JavaProtoCopyTask(task_base.TaskBase):
```

Comment:
Did this task disappear in @lukesneeringer's refactoring? I thought I remembered a task by this name existing in the past.